src/pipeline_v3/utils/security.py

```
@@ -0,0 +1,462 @@
+    return SecretsMasker.mask_log_message(text)
+
+
+def sanitize_input(value: str, input_type: str = "metadata") -> str:
+    """Sanitize user input based on type."""
+    if input_type == "metadata":
+        return InputSanitizer.sanitize_metadata_value(value)
+    elif input_type == "search":
+        return InputSanitizer.sanitize_search_query(value)
+    else:
+        # Default to metadata sanitization for safety
+        return InputSanitizer.sanitize_metadata_value(value)
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)